### PR TITLE
Fix TypeDeclExt/ArrayDeclExt 

### PR DESCRIPTION
**Pull request by douglas-raillard-arm**

### CODE REVIEW

**Comment by inducer** [comment]
Thx!

**Comment by inducer** [comment]
Could you look into the test failures?

**Comment by douglas-raillard-arm** [comment]
Looks like the `align` parameter was added to TypeDecl in 2.21: https://github.com/eliben/pycparser/commit/14ffe7bc486805c568403757d8634211f8458451

I updated the PR with a `pycparser==2.21` version bound

File: setup.py [hunk 1]
```diff
@@ -21,7 +21,7 @@
       python_requires="~=3.8",
       install_requires=[
           "ply>=3.4",
-          "pycparser>=2.18,<=2.20",
+          "pycparser==2.21",
```
Comment (inducer) [inline setup.py:24]: Will this include 2.21.1?

Comment (douglas-raillard-arm) [reply setup.py:24]: It wouldn't, looks like I should use ~= 2.21 instead:
https://peps.python.org/pep-0440/#compatible-release

**Comment by douglas-raillard-arm** [comment]
@inducer PR updated with `~=` instead of `==` in setup.py

**Comment by inducer** [comment]
Thx!

**Pull request merged**